doc/man7/provider-keymgmt.pod

```diff
@@ -219,6 +183,30 @@ returns NULL, the caller is free to assume that there's an algorithm
 from the same provider, of the same name as the one used to fetch the
 keymgmt and try to use that.
 
+=head2 Import and Export Selections
+
+The I<selection> given to OP_keymgmt_import() and OP_keymgmt_export()
+indicates what values of the I<keydata> is of interest.  It can be
+one of:
+
+=over 4
+
+=item B<OSSL_KEYMGMT_WANT_DOMPARAMS>
+
+to indicate that only the domain parameters are interesting.
+
+=item B<OSSL_KEYMGMT_WANT_KEY>
+
+to indicate that only the key data (the data that constitutes the
+private and the public keys) are interesting.
+
+=item B<OSSL_KEYMGMT_WANT_DOMPARAMS>
```

Comment:
??

Reply:
!!

Reply:
(I can do this all day, so maybe you should tell me what you wonder about...)

Reply:
Block copy bug - should be OSSL_KEYMGMT_WANT_BOTH

Reply:
Ah!  Ok, will fix

Reply:
Fixed